src/lib/registration/ui/addon_selection_base_dialog.rb

```diff
@@ -76,12 +86,12 @@ def addon_selected?(_addon)
       # @return [Yast::Term] the main UI dialog term
       def content
         VBox(
-          VStretch(),
           Left(Heading(heading)),
+          Left(CheckBox(Id(:filter_beta), Opt(:notify),
```

Comment:
Moreover the [relevant SCC Trello card](https://trello.com/c/Dj8XvuhA/3323-2-api-of-connect-should-have-information-about-release-state-of-product) says that there might be more states in the future (`alpha`, `pre-alpha`,...). So maybe we should invert the logic and rename the filter to something like `Show only released versions` or `Show only supported versions`...